### PR TITLE
Added padding attribute to TwakeListItem

### PR DESCRIPTION
**Pull request by KhaledNjim**
Added padding attribute to twakeListItem

### CODE REVIEW

File: lib/list_item/twake_list_item.dart [hunk 1]
```diff
@@ -4,21 +4,26 @@ import 'package:linagora_design_flutter/style/linagora_divider_style.dart';
...
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      height: height,
-      decoration: BoxDecoration(
-        border: LinagoraDividerStyle.material().borderDecoration,
+    return Padding(
+      padding: padding ?? EdgeInsets.zero,
```
Comment (nqhhdev) [inline lib/list_item/twake_list_item.dart:19]: Why not use Container `padding` and `margin` properties?

Comment (KhaledNjim) [reply lib/list_item/twake_list_item.dart]: Updated

**Review by nqhhdev — APPROVED** [review]
LGMT

File: lib/list_item/twake_list_item.dart [hunk 2]
```diff
@@ -4,20 +4,26 @@ import 'package:linagora_design_flutter/style/linagora_divider_style.dart';
...
     return Container(
+      margin: margin,
+      padding: padding,
       height: height,
-      decoration: BoxDecoration(
-        border: LinagoraDividerStyle.material().borderDecoration,
-      ),
+        decoration: BoxDecoration(
+          border: LinagoraDividerStyle.material().borderDecoration,
+        ),
```
Comment (hoangdat) [inline lib/list_item/twake_list_item.dart:26]: indent

Comment (KhaledNjim) [reply lib/list_item/twake_list_item.dart]: Updated

**Review by hoangdat — APPROVED** [review]

**Pull request merged**